### ISSUE TITLE
source destination field in TTS evoked from add window

### ISSUE THREAD

**Issue by omega3** [AwesomeTTS/awesometts-anki-addon#77]
This option is available when TTS is evoked from browser:
https://imgur.com/dnXKP3T
but not available when Ctrl+T is pressed in Add window:
https://imgur.com/5tKF5hY

Please, make it available in when Ctrl+T is pressed in Add window.

**Comment by lucwastiaux** [comment]
which option exactly are you missing in the second screen ? can you explain in more detail ?

**Comment by omega3** [comment]
Yes, I know about. Recorded sound is placed where mouse cursor is.  And I use it like this. 

https://imgur.com/5tKF5hY
But I have separate field for sound. Actually, in all note types I have separate field called "Sound". So, each time I need to remember to copy text and place mouse cursor in Sound field. I thought it would be helpful if in the above pop-up would be "Destination Field". This is not something I can't live without but if you are done with more pressing issues this one would be helpful. Maybe as an option (active / inactive), so that it doesn't irritate other users.

**Comment by lucwastiaux** [comment]
I agree that the manual add workflow for adding sound at the same time as adding cards is tedious. we should think of ways to improve that.

**Comment by lucwastiaux** [comment]
There should be a declarative way of telling AwesomeTTS which language is used in what field , and automated generation when adding cards should be an option.

**Comment by lucwastiaux** [comment]
@omega3 language tools has the automation you're looking for: https://languagetools.anki.study/language-tools
i'm planning on doing a re-write of AwesomeTTS which should have similar automation functionality.
